### PR TITLE
fix: await in sequence

### PR DESCRIPTION
**Pull request by DecathectZero**
It's critical that any list of messages coming in are added to the log in the right order.

We have channels/actions that write this incoming data into the database upon adding to the log. 
It can potentially run out of order because we run lines 74-78: https://github.com/logux/core/blob/0a021beaffa01fc5a602ad3d59d35c0cfb8d4708/sync/index.js#L74-L81

the `inFilter` is actually another promise, and on `logux/server`, it awaits `processor.access`:
https://github.com/logux/server/blob/63337798eea393fbeee1c6cb519bce3ae71ef354/server-client/index.js#L214

This means the `this.log.add` and subsequent `processor.process` could be out of order.

### CODE REVIEW

File: sync/index.js [hunk 1]
```diff
@@ -38,11 +38,9 @@ export function sendSynced(added) {
 }
 
 export async function syncMessage(added, ...data) {
-  let promises = []
-
   for (let i = 0; i < data.length - 1; i += 2) {
-    let action = data[i]
-    let meta = data[i + 1]
+    const action = data[i]
```
Comment (ai) [inline sync/index.js:42]: Can you revert this change? It breaks CI since we use another code standard.

Comment (DecathectZero) [reply sync/index.js]: yup, noted

**Comment by ai** [comment]
Good fix. I added `Promise.all` to make it faster by “parallel” processing, but I agree that it could create more problems.

We need to release it as major release. Do you need to release it quickly?

**Comment by DecathectZero** [comment]
> Good fix. I added `Promise.all` to make it faster by “parallel” processing, but I agree that it could create more problems.
> 
> We need to release it as major release. Do you need to release it quickly (or you found some hacky workaround)?

@ai a quicker release would be preferred, but appreciate whatever works best for you!

**Comment by ai** [comment]
I will try to do the release on weekend

**Comment by ai** [comment]
Done. I released Logux Core 0.8 (release name `Ա` in honor of Armenian alphabet) and major versions of Logux Server and Logux Client.

**Pull request merged**